assets/i18n/en.json

```diff
@@ -567,6 +569,7 @@
     "I18N_LEARN_TOPIC": "Learn <[topicName]>",
     "I18N_LESSON_AUTHORS_DROPDOWN_TITLE": "Lesson Authors",
     "I18N_LESSON_INFO_HEADER": "Lesson Info",
+    "I18N_LESSON_INFO_TOOLTIP_MESSAGE": "Great job on reaching your second checkpoint! View your progress and other lesson information here.",
```

Comment:
You've reached a checkpoint. Great job! View your progress and other lesson information here.

Reply:
Done